tests/profile/profile-page.spec.ts

```diff
@@ -68,7 +68,7 @@ test.describe('Profile Page - Basic display tests', () => {
 test.describe('Profile Page - Lists', () => {
   test.describe.configure({ mode: 'serial' });
 
-  test(`Facets appear`, async ({ browser }) => {
+  test.fixme(`Facets appear`, async ({ browser }) => {
```

Comment:
`fixme` here - does this test run?

Reply:
tests with `fixme` annotations will be skipped